### ISSUE TITLE
Complience with Mozilla guidelines

### ISSUE THREAD

**Issue by MatthiasSaihttam** [ka-extension/ka-extension-ts#190]
The KA Extension was recently removed from the Mozilla webstore for the following issues:

- Including remote Ace scripts from JS Delivr [here](https://github.com/ka-extension/ka-extension-ts/blob/abf37ec5552be1222650a49d313dfb5f8b1fa2af/src/project.ts#L125)

- "Creating DOM nodes from HTML strings containing potentially unsanitized data", [here](https://github.com/ka-extension/ka-extension-ts/blob/abf37ec5552be1222650a49d313dfb5f8b1fa2af/src/profile.ts#L85), [here](https://github.com/ka-extension/ka-extension-ts/blob/abf37ec5552be1222650a49d313dfb5f8b1fa2af/src/popup.ts#L221), and "probably others"

Additionally, they are asking for a privacy policy.

I have no plans to address these problems at this time.

**Comment by JettBurns14** [comment]
I don't recall working on the Ace scripts feature, so not sure how that'd be fixed.

Regarding unsanitized data, perhaps we can add a general checker/filtering function every time we inject data into HTML. I can imagine users getting creative and possibly getting an alert popup if they really tried, but I haven't heard of any issues coming from this design of ours.

I'm fine writing a privacy policy, if we fix the other issues. I welcome any PRs which get the ball rolling on these things!

I'd like to get the extension back on Firefox one day... but I'm busy with other things currently. Would love to get back into this project if I get some time!

**Comment by MatthiasSaihttam** [comment]
Yeah, there's no security issue with our implementation, our data is sanitized, it's just an issue of best practice. They would like us to use DOM manipulation instead of creating the elements from HTML strings.
